database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -17,6 +17,7 @@ public function up()
             $table->string('name');
             $table->string('email')->unique();
             $table->string('password');
+            $table->integer('type');
```

Comment:
How about renaming it to 'role'? I find it less confusing and in case we ever needed to list all the support agents we will just create a Role model and then call Role.users. 
What do you think?